client/tests/webdriver/specs/onboardNewUser.spec.js

```diff
@@ -1,11 +1,18 @@
 import { expect } from "chai"
-import OnboardPage from "../pages/onboard.page"
-
 import CreatePerson from "../pages/createNewPerson.page"
+import OnboardPage from "../pages/onboard.page"
+import { createOnboardingNewPerson } from "./newUserUtils"
 
 describe("Onboard new user login", () => {
+  it("Should successfully create an account", () => {
+    // unique name if we want to run tests without resetting DB
+    OnboardPage.openAsOnboardUser("/", `bonny${Date.now()}`)
```

Comment:
The reason for `Date.now()` is to solve this situation: 
- You start the test server/DB
- You run the tests
- Now you have a "bonny" in the DB which is not a onboard user anymore
- I can't run the same test without resetting the DB